src/pygama/lgdo/table.py

```diff
@@ -173,5 +175,6 @@ def get_dataframe(self, cols=None, copy=False):
         for col in cols:
             if not hasattr(self[col],'nda'):
                 print(f'column {col} does not have an nda, skipping...')
-            else: df[col] = self[col].nda
+            else: df[col] = list(self[col].nda)
```

Comment:
I think the conversion to a list is unnecessary, right? It should be removed because it takes CPU + memory.